### PR TITLE
Use primereact for View Sequence dialog

### PR DESCRIPTION
**Pull request by toddburnside**

### CODE REVIEW

**Comment by cquiroz** [comment]
Screenshots?

**Comment by toddburnside** [comment]
It doesn't look a lot different. Except for the cheerful button, of course.
image

File: common/src/main/scala/explore/components/ui/ExploreStyles.scala [hunk 1]
```diff
@@ -371,6 +371,9 @@ object ExploreStyles {
   val TargetSummarySubRowCell: Css = Css("explore-target-summary-subrow-cell")
   val ConstraintsSummaryEdit: Css  = Css("explore-constraints-summary-edit")
 
+  object Dialog: // SUI has Mini, Tiny, Small, Large and Fullscreen.
+    val Small: Css = Css("pl-dialog-small")
```
Comment (cquiroz) [inline common/src/main/scala/explore/components/ui/ExploreStyles.scala:375]: Should these go to `PrimeStyle` eventually?

File: explore/src/main/scala/explore/config/SequenceEditorPopup.scala [hunk 2]
```diff
@@ -36,41 +35,38 @@ object SequenceEditorPopup:
   private val component =
     ScalaFnComponent
       .withHooks[Props]
-      .useState(false)        // isOpen
+      .useStateView(false)    // isOpen
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/SequenceEditorPopup.scala:38]: I've been trying to switch these to use `NewTypes` to make them more readable

**Review by cquiroz — APPROVED** [review]
LGTM

File: common/src/main/scala/explore/components/ui/ExploreStyles.scala (hunk 1 above)
Comment (toddburnside) [reply common/src/main/scala/explore/components/ui/ExploreStyles.scala:375]: I don't know. The CSS depends on the breakpoints we set in explore. It doesn't seem like the `Css` should go there if it isn't backed by actual styles.

File: explore/src/main/scala/explore/config/SequenceEditorPopup.scala (hunk 2 above)
Comment (toddburnside) [reply explore/src/main/scala/explore/config/SequenceEditorPopup.scala:38]: Sure, I can switch those in the morning.

File: common/src/main/scala/explore/components/ui/ExploreStyles.scala (hunk 1 above)
Comment (toddburnside) [reply common/src/main/scala/explore/components/ui/ExploreStyles.scala:375]: It's possible we could move it all to Lucuma UI to be used across lucuma projects.

**Pull request merged**